build/npm-run.py

```diff
@@ -1,10 +1,18 @@
 #!/usr/bin/env python
 import os
+import subprocess
 import sys
 
 SOURCE_ROOT = os.path.dirname(os.path.dirname(__file__))
-args = ["npm", "run",
+cmd = "npm"
+if sys.platform == "win32":
+    executable += ".cmd"
```

Comment:
`executable` is undefined.  Did you mean:
```suggestion
    cmd += ".cmd"
```